src/procdialogs.cpp

```diff
@@ -164,11 +184,21 @@ procdialog_create_renice_dialog (ProcData *procdata)
     if (renice_dialog)
         return;
 
+    gtk_tree_selection_selected_foreach (procdata->selection, get_last_selected,
+                                         &info);
+    gint selected_count = gtk_tree_selection_count_selected_rows (procdata->selection);
+
     if (!info)
         return;
 
-    dialog_title = g_strdup_printf (_("Change Priority of Process “%s” (PID: %u)"),
-                                    info->name, info->pid);
+    if ( selected_count == 1 ) {
+        dialog_title = g_strdup_printf (_("Change Priority of Process “%s” (PID: %u)"),
+                                        info->name, info->pid);
+    } else {
+        dialog_title = g_strdup_printf (_("Change Priority of %d selected processes"),
```

Comment:
Use title case, .i.e `Change Priority of %d Selected Processes`

Reply:
done